main.development.js

```diff
@@ -15,7 +17,49 @@ let menu;
 let template;
 let mainWindow = null;
 
-// const clearLog = () => fs.writeFile(OPTIONS.logpath, '');
+const clearLog = () => fs.writeFile(OPTIONS.logpath, '');
+
+const logToFile = bunyan.createLogger({
+    name: 'plotly-database-connector-logger',
+    streams: [
+        {
+            level: 'info',
+            path: OPTIONS.logpath
+        }
+    ]
+});
+
+function log(logEntry, code = 2) {
+
+    // default log detail set to 1 (warn level) in ./args.js
+    if (code <= OPTIONS.logdetail) {
+        switch (code) {
+            case 0:
+                logToFile.error(logEntry);
+                break;
+            case 1:
+                logToFile.warn(logEntry);
+                break;
+            case 2:
+                logToFile.info(logEntry);
+                break;
+            default:
+                logToFile.info(logEntry);
+        }
+
+        if (!OPTIONS.headless) {
+            mainWindow.webContents.send(channel, {
+                log: {
+                    logEntry,
+                    timestamp: timestamp()
+                }
+            });
+        }
+
+    }
+}
```

Comment:
I think this should be in like a `logger.js` file instead of in `main.development.js`